style.css

```diff
@@ -1,336 +1,764 @@
+    margin-right: 100px;
+  }
 
-.ab-li-1 {
-  margin-top: 30px;
-  margin-left: 100px;
-  font-weight: 700px;
-  font-size: 25px;
-}
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 25px;
+  }
 
-.ab-text {
-  font-size: 21px;
-  line-height: 28px;
-  text-align: center;
-  font-weight: 400;
-  padding: 19px;
-  color: #42526e;
-}
+  .card-flex {
+    height: 474px;
+    background-color: #ebf0ee;
+    border: #cfd0d1 solid 1px;
+    border-radius: 8px 8px 8px 8px;
+  }
 
-.ab-btn {
-  background: #fff;
-  color: #36b37e;
-  display: flex;
-  padding-left: 15px;
-  padding-right: 10px;
-  padding-top: 10px;
-}
+  .description {
+    height: 225px;
+    margin-top: 249px;
+    border-radius: 8px 8px 8px 8px;
+    background: #fff;
+  }
 
-.flex-tool {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  align-items: center;
-  justify-content: center;
-}
+  .new {
+    color: #3a4a42;
+    text-align: center;
+    padding-top: 15px;
+    font-size: 26px;
+    line-height: 32px;
+  }
 
-.card-flex {
-  width: 320px;
-  height: 474px;
-  background-color: #ebf0ee;
-  border: #cfd0d1 solid 1px;
-  border-radius: 8px 8px 8px 8px;
-}
+  .p-lang {
+    padding: 0;
+    display: flex;
+    justify-content: center;
+  }
 
-.ab-btn-btn {
-  margin-right: 90px;
-  width: 160px;
-  margin-left: 0;
-  padding-left: 10px;
-  margin-bottom: 12px;
-}
+  .btn-btn {
+    height: 35px;
+    font-size: 15px;
+    border-radius: 4px 4px 4px 4px;
+    margin: 3px;
+    margin-top: 10px;
+    color: #3a4a42;
+    background: #ebf0ee;
+    border: none;
+  }
 
-.ab-lang {
-  background: #ebf0ee;
-  height: 352px;
-  margin-top: 50px;
-  margin-left: 25px;
-}
+  .btn {
+    height: 50px;
+    background: #36b37f;
+    border-radius: 6px 6px 6px 6px;
+    border: none;
+    display: flex;
+    color: #fff;
+    font-size: 19px;
+    padding-top: 14px;
+    padding-left: 13px;
+  }
 
-.ab-li {
-  padding: 0;
-}
+  .sec2-btn {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 20px;
+  }
 
-.ab-title {
-  margin-top: 10px;
-  padding-top: 0;
-  color: #344563;
-}
+  /* ************************ section Two Above **************** */
+  .mob-footer-img {
+    display: none;
+  }
 
-.ab-head {
-  margin-top: 60px;
-  padding-top: 0;
-  color: #344563;
-  font-size: 40px;
-  margin-left: 100px;
-}
+  .mob-img {
+    display: none;
+  }
 
-.ab-img {
-  margin-top: 20px;
-}
+  .about {
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 100px;
+    margin-bottom: 50px;
+  }
 
-.close {
-  margin-left: 25px;
-  margin-top: 50px;
-  background: #fff;
-  border: none;
-}
+  .about-li {
+    display: flex;
+    flex-direction: column;
+  }
 
-.closure {
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-right: 25px;
-}
+  .ab-head {
+    color: #344563;
+    font-size: 40px;
+    margin-left: 80px;
+    margin-left: 200px;
```

Comment:
:red_circle: [OPTIONAL] A close look at the Figma design shows that this title is centered. Kindly remove this `margin-left` property and set `text-align: center;` for this element.

Your Design
![ab-sec](https://user-images.githubusercontent.com/26736582/200121104-b87ae947-9263-46a2-b14c-d5ce70d61ba0.png)

Figma Design
![about-me](https://user-images.githubusercontent.com/26736582/200121227-1d12f437-0c63-4a93-9430-42e36d3ce3f3.png)